### DISCUSSION TITLE
how to get the "epoch" value in the "validate_epoch_end"

### DISCUSSION THREAD

**Discussion by emilyemliyM** [Lightning-AI/pytorch-lightning#13543] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
how to get the "epoch" value in the "validate_epoch_end" 

Because I wanna use the "epoch" value as a  output datadir parameter.

thank a lot.

**Comment by Struggle-Forever** [comment, marked as answer]
you can get  epoch using self.current_epoch
